lte/gateway/python/magma/pipelined/tests/test_enforcement_stats.py

```diff
@@ -679,6 +680,56 @@ def test_rule_reactivation(self):
         #                 num_pkts_tx_match * len(packet))
         self.assertEqual(len(stats), 2)
 
+ 
+    #additional test for stats enforcement
+    def test_cookie_poll(self):
+        original = self.enforcement_stats_controller._poll_stats
```

Comment:
Add comment
        """
        Add a subscriber policy, verify flows are properly installed
        Assert:
            Query with RULE_NUM 1 returns proper values
        """

Reply:
The comment block should be inside of the function
```
def test_cookie_poll(self):
    ... comment goes here
```